junit/kube-api-test/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2015 Red Hat, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.fabric8</groupId>
+    <artifactId>kubernetes-client-project</artifactId>
+    <version>6.11-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+
+    <artifactId>kube-api-test-parent</artifactId>
+
+    <packaging>pom</packaging>
+
+    <name>Kube API Test Parent</name>
+    <description>Library to support testing against Kubernetes API server</description>
+
+    <properties>
+        <maven.compiler.release>11</maven.compiler.release>
+        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
Do we really need this module to be on JDK11 ? Most of the modules in this repository are based on JDK8. If not, we would also need to modify [release script](https://github.com/fabric8io/kubernetes-client/blob/main/.github/workflows/release.yaml#L100) to add this as list of java 11 modules to be released.

Reply:
this was discussed at the internal meeting. It uses the JDK HTTP client.

Reply:
@manusa : Do we need to modify release script to include this module in `Build and release Java 11 modules` step?